src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -232,7 +232,11 @@ public synchronized void setDeadlineTask(int indexInFilteredList, DeadlineTask n
     }
 
     @Override
+    /**
+     * Return the filtered deadline task list, all finished deadline task will be filtered out
+     */
```

Comment:
hmm, checkstyle didn't complain about this? Usually the javadoc should be before the annotation.